### DISCUSSION TITLE
Roadmap for actions.github.com/v1alpha1 and actions.summerwind.net/v1alpha1

### DISCUSSION THREAD

**Discussion by roshitho** [actions/actions-runner-controller#2560] [Questions]
As per the current github doc [here](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/about-actions-runner-controller).

_

> You can find the list of custom resource definitions used for ARC in the following API schema definitions.
> [actions.github.com/v1alpha1] and [actions.summerwind.net/v1alpha1].

_

But they are not same and not interchangeable. This raises a confusion, going forward are both going to be maintained? If not which one should be used for github cloud organizations. 

We have tested horizontal scaling with metrices like PercentageRunnersBusy etc. What is the equivalent settings for actions.github.com/v1alpha1 - AutoscalingListener/AautoscalingRunnerset.

**Comment by pashcan** [comment]
I second your sentiment as it's been somewhat confusing to get started with ARC in its current state of flux.
You already know this since the [maintainer replied to your question in another thread](https://github.com/actions/actions-runner-controller/discussions/2174#discussioncomment-5836259), but it might be useful for others finding this thread:
> `actions.github.com/v1alpha1` is the path forward

Personally I'm leaning towards setting things up using the "old" approach until the dust and documentation settles with the new scale set.
